.drone/scripts/build-native.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+# Setup
+git fetch
+
+export VERSION=$(cat .data.json | jq -r '.current_pkgver')
+
+# Tar current directory
+tar -cJf ../makedeb_$VERSION.orig.tar.xz .
+
+# Build makedeb.
+debuild -us -uc
```

Comment:
Since the script's running so few commands I'd just remove the comments as it should be fairly self-explanatory:

```sh

#!/usr/bin/env bash
set -e

git fetch
export VERSION=$(cat .data.json | jq -r '.current_pkgver')
tar -cJf ../makedeb_$VERSION.orig.tar.xz .
debuild -us -uc
```